app/src/main/res/layout/fragment_child_monitor.xml

```diff
@@ -114,23 +114,33 @@
             />
 
     <TextView
+            style="@style/ShadowTextViewStyle"
             android:id="@+id/timer"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:layout_marginStart="@dimen/activity_horizontal_margin"
             android:layout_marginTop="@dimen/activity_vertical_margin"
             android:layout_marginEnd="@dimen/activity_horizontal_margin"
-            android:shadowColor="@android:color/black"
-            android:shadowDx="4"
-            android:shadowDy="4"
-            android:shadowRadius="2"
             android:textSize="16sp"
             app:layout_constraintEnd_toEndOf="@+id/surfaceView"
             app:layout_constraintStart_toStartOf="@+id/surfaceView"
             app:layout_constraintTop_toTopOf="@+id/surfaceView"
             tools:text="4:59"
             />
 
+    <co.netguru.baby.monitor.client.feature.debug.DebugView
```

Comment:
shoudn't we have this visible only in debug builds? I don't see any logic that disables this feature.

Reply:
This won't do anything. This view to get data uses class that has empty implementation in release build